EssentialsDiscord/src/main/java/net/essentialsx/discord/DiscordSettings.java

```diff
@@ -298,6 +326,11 @@ private MessageFormat generateMessageFormat(String content, String defaultStr, b
 
     @Override
     public void reloadConfig() {
+        if (plugin.isInvalidStartup()) {
+            plugin.getLogger().warning("Tried to reload EssentialsX Discord config while the plugin is in an invalid state!");
```

Comment:
Might want to show a message about "if you've modified your config, restart your server"